recipes/python-toolchain/activate.bat

```diff
@@ -0,0 +1 @@
+set PIP_CONFIG_FILE="%LIBRARY_PREFIX%\etc\pip.conf"
```

Comment:
Good point. I have seen that syntax for unsetting a variable, but not setting it. Will change.